@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -59,30 +60,32 @@ export class TileEntry {
 }
 
 /**
- * Missing Typedoc
+ * Represents a unique TileKey and the area it takes up on screen.
  */
 class TileKeyEntry {
-    constructor(public tileKey: TileKey, public area: number) {}
+    constructor(public tileKey: TileKey, public offset: number, public area: number) {}
 }
 
 /**
- * Missing Typedoc
+ * Stores Tile's or TileProxy's in an LRUCache
```

Comment:
Stores [[Tiles]]s or [[TileProxy]]s in the [[LRUCache]].

Reply:
Thanks